documentation/book/assembly-tls-sidecar.adoc

```diff
@@ -0,0 +1,26 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-tls-sidecar-{context}']
+
+= TLS Sidecar
+
+Sidecar is a container which is running in a pod and serves an auxiliary purpose.
+In case of TLS sidecar, the purpose is encrypting / decrypting the communication between {ProductName} componenets and Zookeeper.
```

Comment:
- componenets > components
- the purpose is encrypting / decrypting the communication between  > the purpose is to encrypt or decrypt the communication between